docs/bar.js.html

```diff
@@ -270,6 +270,7 @@ <h1 class="page-title">Source: bar.js</h1>
             xTicks = 5,
             percentageAxisToMaxRatio = 1,
             enablePercentageLabels = false,
+            padding = 0.1,
```

Comment:
Let's rename this to something a bit more specific, as it could be confused with a 'general' chart padding.

Maybe `betweenBarsPadding` or `barPadding`?